docs/source/ml.rst

```diff
@@ -77,6 +77,21 @@ method. If the class you're using doesn't have a ``predict_proba`` method,
 you can add one by wrapping it in a :class:`~sklearn.calibration.CalibratedClassifierCV`.
 
 
+Custom Dependencies
+-------------------
+
+Installing packages from PyPI is straightforward. You can specify a `dependencies`
+argument to `~civis.ml.ModelPipeline` which will install the dependencies in your runtime
```

Comment:
I'd give an explicit example to show what this looks like. And you should strongly encourage users to peg versions of dependencies. If they don't, then there's a serious risk of scoring jobs failing. It's probably worth mentioning both here and in the doc string.